electrical/readme.MD

```diff
@@ -0,0 +1,261 @@
+   
+**Table 7. Wire Lengths by Section**
+
+Section | Wires Needed | Length [inches]
+------- | ------------ | --------------- 
+A | 2x | 20AWG & 4x 30AWG | 35
+B | 2x | 20AWG & 4x 30AWG | 28
+C | 2x | 20AWG & 4x 30AWG | 48
+D | 2x | 20AWG & 3x 30AWG | 20
+E | 2x | 20AWG & 3x 30AWG | 32
+
+![Wire Lengths](../images/electronics/wire_labels.png)
+
+  **Figure 8. Wire lengths**
+  
+## 4.1. Drive Motors
+We will begin with the drive motors (the 6 motors that are directly connected to each drive wheel). Make sure to look at which RoboClaw corresponds to which motors that that RoboClaw controls. It is important that this ordering is preserved. Figure 9 shows the motor pinout for the motors for the drive system. You will be wiring each of these to the terminal blocks on the control board. One method of doing this is to cut off the 6 pin connector from the motor and solder directly to the wires there. Another way is to add your own connector if you wish to be able to change the motor out easier in the future.
+
+![Roboclaw Motor Assignments](../images/electronics/motor_id.png)
+
+**Figure 9: RoboClaw - Motor Assignments**
+  
+The terminal blocks that correspond to the drive motors are connectors J17-J22. You will see them labeled on the control board corresponding to which motor it should connect to1.
+Each of the terminal blocks are identical for the drive motors; pay attention to the silkscreen to know which screw terminal to plug the wire into. The terminal block position on the board will correspond roughly with which motor it goes to on the robot, ie. the J17 connector is in the front-right of the
+board when installed in the robot, and connects to the front-right drive wheel.
+
+**Table 8. Drive Motor and Encoder Wiring**
+
+Control Board Label | Motor Wire Color | Signal
+------------------- | ---------------- | ------
+Mx+ | Red | Motor+
+Mx- | Black | Motor-
+GND + | Green | Ground
+5V | Blue | Encoder Power
+ENxA | Yellow | Encoder signal A
+ENxB | White | Encoder signal B
+
+
+## 4.2. Corner Motors/Encoders
+The big difference between the corner motors and drive motors is that the corners use absolute encoders. The absolute encoders are separate from the motor and so we have to do a little bit of work to get them to interface with the motor controllers. The corner motors only have the two connections to the physical motor: the (+) and (-) going to the top of the motor. However, you must also hook up the absolute encoder. When plugged into the encoder, the 3-pin Micro connector will have 3 wires coming out of it, shown in Figure 10.
+
+*Make sure you are using a REV-E or newer board. There was an initial release of board files that had the silkscreen reversed from left-right for all motors (drive and corner). If you have a REV-D or earlier board, the silkscreen may have these labels backwards.*
+
+![Encoder connection](../images/electronics/3_pin_encoder.png)
```

Comment:
scale this image down a bit, use html like `<img src="/images/body/9x12_Plate_cut2.png" height="350">`

see https://github.com/nasa-jpl/open-source-rover/blob/master/examples/README_example.md

Reply:
I left these big because they are helpful when running cabling.